app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+                    table.row
+                      tbody
+                        tr
+                          th.align-middle.small-12.large-12.columns.first.last
+                            table
+                              tr
+                                th
+                                  = image_tag("logo.png", class: "small-float-center")
+                                th.expander
+                    table.row
+                      tbody
+                        tr
+                          th.small-12.large-12.columns.first.last
+                            table
+                              tr
+                                th
+                                  table.spacer
+                                    tbody
+                                      tr
+                                        td height="35px" style="font-size:35px;line-height:35px;"   
+                                  h4
+                                    b
+                                      = message.subject
+                                  hr
+                                  p.lead 
+                                    == yield
+                                  hr
+                                  p
+                                    i Please do not reply to this message.
+                                  table.spacer
+                                    tbody
+                                      tr
+                                        td height="35px" style="font-size:35px;line-height:35px;"   
+                                th.expander
+                    table.wrapper.footer align="center" 
+                      tr
+                        td.wrapper-inner
+                          table.row
+                            tbody
+                              tr
+                                th.small-12.large-12.columns.first.last
+                                  table
+                                    tr
+                                      th
+                                        p 
+                                          'For more help, please contact the #{APP_NAME} Customer Contact Center via web form at 
+                                          = link_to nil, Figaro.env.support_url
+                                      th.expander
+                    table.wrapper.legal align="center" 
+                      tr
+                        td.wrapper-inner
+                          table.row
+                            tbody
+                              tr
+                                th.small-12.large-12.columns.first.last
+                                  table
+                                    tr
+                                      th
+                                        p.text-center
+                                          a href="#" About #{APP_NAME} 
+                                          ' | 
+                                          a href="#" Privacy policy
```

Comment:
Are these links placeholders for now or do we have About and Privacy policy pages we can link to?